DATA/UNIVERSE/SYSTEMS/Br04/Br04.ini

```diff
@@ -1542,3464 +1605,3594 @@ faction = co_be_grp, 0.38
-density = 3
-repop_time = 25
-max_battle_size = 4
-pop_type = Background
-relief_time = 35
-encounter = area_bh_scout, 7, 1
-faction = gd_bh_grp, 1
+[Object]
+nickname = Br04_hazard_buoy_32
+ids_name = 261163
+pos = -40343, 0, -75680
+archetype = hazard_buoy
+ids_info = 66144
 
-[zone]
-nickname = Zone_Br04_path_bowex1_1
-pos = 4642, 0, -51273
-rotate = 90, -87, 180
-shape = CYLINDER
-size = 750, 45843
-sort = 96
+[Object]
+nickname = Br04_space_tanklx4_dmg_1
+pos = 47174, 200, -19036
+rotate = 20, 10, 160
+archetype = space_tanklx4_dmg
+
+[Object]
+nickname = Br04_space_beaml_dmg_1
+pos = 47164, 250, -18797
+rotate = -14, 0, 0
+archetype = space_beaml_dmg
+
+[Object]
+nickname = Br04_space_tankl_3dmg_1
+pos = 47167, 400, -18665
+rotate = 33, -16, 180
+archetype = space_tankl_3dmg
+
+[Object]
+nickname = Br04_space_tankl_1dmg_1
+pos = 54148, 250, -26833
+archetype = space_tankl_1dmg
+
+[Object]
+nickname = Br04_space_tankl_1dmg_2
+pos = 54178, 250, -26920
+rotate = 0, 180, 0
+archetype = space_tankl_1dmg
+
+[Object]
+nickname = Br04_space_tankl_1dmg_3
+pos = 54124, 243, -26766
+archetype = space_tankl_1dmg
+
+[Object]
+nickname = Br04_space_tankl_2dmg_1
+pos = 54436, -242, -4163
+rotate = 30, 30, 30
+archetype = space_tankl_2dmg
+
+[Object]
+nickname = Br04_space_habitat_dmg_1
+pos = 54273, -249, -4106
+rotate = 0, 0, 120
+archetype = space_habitat_dmg
+
+[Object]
+nickname = Br04_space_industrial_dmg_1
+pos = 64758, 250, 5224
+rotate = 20, 20, 20
+archetype = space_industrial_dmg
+
+[Object]
+nickname = Br04_space_tankl_3dmg_2
+pos = 70408, 100, 1613
+rotate = 40, 40, 40
+archetype = space_tankl_3dmg
+
+[Object]
+nickname = Br04_space_beamx_dmg_1
+pos = 71061, -300, -34976
+rotate = 80, 0, 80
+archetype = space_beamx_dmg
+
+[Object]
+nickname = Br04_space_beaml_dmg_2
+pos = 78098, -300, -22051
+rotate = 26, 10, 100
+archetype = space_beaml_dmg
+
+[Object]
+nickname = Br04_space_tanks_1dmg_1
+pos = 77342, 400, -10940
+rotate = 10, 20, 50
+archetype = space_tanks_1dmg
+
+[Object]
+nickname = Br04_space_tankl4x4_dmg_1
+pos = 52935, -300, -14325
+rotate = 20, 15, -20
+archetype = space_tankl4x4_dmg
+
+[Object]
+nickname = Br04_space_tankl4_dmg_1
+pos = 52804, -250, -14272
+rotate = -150, 80, 180
+archetype = space_tankl4_dmg
+
+[Object]
+nickname = Br04_ast_solar_mineable_60_1
+pos = 69853, 220, -20327
+archetype = ast_solar_mineable_60
+
+[Object]
+nickname = Br04_ast_solar_mineable_60_2
+pos = 70574, 60, -11423
+rotate = -75, 80, 0
+archetype = ast_solar_mineable_60
+
+[Object]
+nickname = Br04_suprise_bw_fighter_1
+ids_name = 261721
+pos = -25288, -500, 47727
+rotate = -5, -16, 60
+archetype = suprise_bw_fighter
+visit = 16
+ids_info = 66404
+loadout = SECRET_ou_bw_fighter_br04a
+
+[Object]
+nickname = Br04_surprise_bw_fighter_2
+ids_name = 261722
+pos = -38188, 700, -70465
+rotate = -27, -12, -5
+archetype = suprise_bw_fighter
+visit = 16
+ids_info = 66405
+loadout = SECRET_ou_bw_fighter_br04b
+
+; Ambient Zones [Must Come First]

```

Comment:
This is actually wrong. Ambient zones must always come last due how their density and population_additive flag work together.